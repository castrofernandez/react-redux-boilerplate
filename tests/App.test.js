import React from 'react';
import App from '../src/js/App';
import { create } from 'react-test-renderer';

describe('First test suite', () => {
  test('First test', () => {
    let tree = create(<App />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});