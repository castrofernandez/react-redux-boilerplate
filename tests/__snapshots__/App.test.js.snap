// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First test suite First test 1`] = `
Array [
  <header
    className="header"
  >
    App Header
  </header>,
  <section
    className="body"
  >
    <h1
      className="title"
    >
      React+Redux boilerplate
    </h1>
  </section>,
  <footer
    className="header"
  >
    App Footer
  </footer>,
]
`;
